DockerFile

```diff
@@ -0,0 +1,46 @@
+FROM python:3.8-slim-buster
+
+RUN apt-get update
+RUN apt-get -y install gcc
+
+WORKDIR /rest/BackendApp
+
+COPY ./requirements/base.txt ./
+COPY ./requirements/local.txt ./
+COPY ./requirements/production.txt ./ 
+COPY ./requirements/staging.txt ./
+
+# Prevents Python from writing pyc files to disc
+ENV PYTHONDONTWRITEBYTECODE 1
+
+# Prevents Python from buffering stdout and stderr
+ENV PYTHONUNBUFFERED 1
+
+
+RUN pwd
+
+RUN ls -l
+
+RUN ls -a
+
+COPY ./docker-build/wsgi-entry.sh /rest/
+
+RUN chmod +x /rest/wsgi-entry.sh 
+
+RUN  apt-get install -y --no-install-recommends python-dev libpq-dev
+
+RUN mkdir -p /rest/logs && \
+    pip3 install wheel==0.34.2 && \
+    pip3 install -r base.txt 
+#    pip3 install -r local.txt && \
+#    pip3 install -r production.txt && \
+#    pip3 install -r staging.txt 
```

Comment:
well for this.....we can get the platform env and only run a single install......for instance local.txt contains the django debug toolbar....that is not required for production

Reply:
Ideally, we shud have only a single requirements.txt. Or just 2, one for UAT another for Prod.